code/modules/power/lighting.dm

```diff
@@ -843,6 +845,14 @@
 	sleep(1)
 	qdel(src)
 
+/obj/machinery/light/proc/on_light_eater(obj/machinery/light, datum/light_eater)
+	SIGNAL_HANDLER
+	if(status == 1)
```

Comment:
What is `status == 1`? Could you fix this magic number?

Reply:
Sure. I think I just copied this from the old proc.

Reply:
Done